Core/SymbolValidation/SymbolValidator.cs

```diff
@@ -46,15 +48,7 @@ public async Task<SymbolValidatorResult> Validate(CancellationToken cancellation
         {
             try
             {
-                // NuGet signs all its packages and stamps on the service index. Look for that.
-                if (_package is ISignaturePackage sigPackage)
-                {
-                    await sigPackage.LoadSignatureDataAsync().ConfigureAwait(false);
```

Comment:
Why is the signature load removed -- I didn't see it called elsewhere?

Reply:
It has been too long for me to remember. I am sure I thoroughly debugged the code and there was a good reason for it (e.g. MyGet would not work otherwise. Do you want me to check/debug again why this change was necessary?